packages/light-apps/src/Content/index.tsx

```diff
@@ -41,7 +48,13 @@ const ID_CARD_ACTIONS = (name: string) => {
 
 // @ts-ignore
 @(withRouter as any)
+@inject('onboardingStore')
+@observer
 class Content extends React.Component<Props> {
+  get injected () {
```

Comment:
Why do you need this?

`      const { onboardingStore: { isFirstRun } } = this.props; ` should work

Reply:
Ah, sorry, just saw the resource. I'll have a look

Reply:
I actually disagree with the article, this is too hacky.

If you don't want to do `variable && variable.subVariable`, use `variable!.subVariable`. https://stackoverflow.com/questions/42273853/in-typescript-what-is-the-exclamation-mark-bang-operator-when-dereferenci